app/assets/stylesheets/views/_landing_page/hero.scss

```diff
@@ -1,25 +1,32 @@
 @import "govuk_publishing_components/individual_component_support";
 
 .app-b-hero {
-  margin-bottom: govuk-spacing(4);
+  margin-bottom: govuk-spacing(6);
 }
 
+.app-b-hero__imagewrapper {
+  display: block;
+  min-height: 200px;
```

Comment:
We should probably have width / height attributes on each of the sources / the img inside the picture element, so that the browser doesn't have to wait for the stylesheet to make an educated guess about the size of the images.

If we had that, you probably wouldn't need this?

Reply:
That's a good point, but it sort of depends on how we want to handle the image. If we just set the height then it breaks the aspect ratio at a lower width:

![Screenshot 2024-10-09 at 16 00 44](https://github.com/user-attachments/assets/0bd6c82b-0170-441c-b4eb-94b24de313bf)

However, we can fix the height and centre the image in the space, but that would crop the edges off at a lower width:

![Screenshot 2024-10-09 at 16 04 10](https://github.com/user-attachments/assets/bc87f710-a70f-48b6-b6c5-9fd832652d8c)

Think this might be a question for design.

Reply:
Can we set the width and the height to the intrinsic size of the image, and then override them in CSS once that's loaded?

I think that way we'd get a flash of the image being not-quite-the-right size (but not stretched or cropped) before the CSS loads and right-sizes it. Which is better than a flash of the image being totally the wrong size.

Reply:
@richardTowers have reworked this block to account for the height suggestion, and updated PR description to explain.

Reply:
Looks good as far as I can tell. I'll resolve this conversation, but I think I'd want a frontender to actually review the CSS.

Reply:
I appreciate this has already been resolved, but this looks like a candidate for `aspect-ratio` and `object-fit` CSS properties. Those could provide a better overall solution with more flexibility for different sized images. If the aspect-ratio of the image container is determined in the CSS, eg `aspect-ratio: 16 / 9` and we use `object-fit:cover` on the image, any image that is too large will automatically be cropped within the container but retain its aspect ratio. Is that the objective here?